assets/js/components/Button.stories.js

```diff
@@ -30,6 +30,8 @@ DefaultButton.args = {
 	children: 'Default Button',
 	href: '#',
 };
+// Test empty scenario to create new VRT.
+DefaultButton.scenario = {};
```

Comment:
I added this to test that the code populated the label correctly.

New VRTs have been added

I can of course remove this before merging (or remove after on a new ticket) or even include another new VRT that may be needed